docs/quickstart.rst

```diff
@@ -34,14 +34,18 @@ Using Web3
 ----------
 
 To use the web3 library you will need to initialize the
-:class:`~web3.Web3` class.
+:class:`~web3.Web3` class by connecting to an Ethereum node. The quickest way to do so for free is by setting up an account on `Infura https://infura.io/>`. On Infura, create a project and copy the Project ID. Then set the environment variable ``WEB3_INFURA_PROJECT_ID``.
```

Comment:
I would say here: `class and connect to an Ethereum node... `. It's not quite accurate to say initialize the class _by_ connecting to the node, because you can initialize the class without connecting to a node.

Reply:
Agree, my wording is incorrect, that makes more sense

Reply:
fixed with commit d9b744a